services/tests.py

```diff
@@ -1,3 +1,120 @@
-from django.test import TestCase
+from django.test    import TestCase, Client
 
-# Create your tests here.
+from django.test.client import RequestFactory
+
+
+from unittest.mock  import patch, MagicMock
+
+from services.models        import *
+
+import json
+
+client = Client()
+class CategoryViewTest(TestCase):
+
+    def setUp(self):
+        t1 = Category.objects.create(
+            name           = 'Test1',
+            icon_image_url = 'test_icon.url1'
+        )
+        t2 = Category.objects.create(
+            name           = 'Test2',
+            icon_image_url = 'test_icon.url2'
+        )
+        t3 = Category.objects.create(
+            name           = 'Test3',
+            icon_image_url = 'test_icon.url3'
+        )
```

Comment:
```suggestion
        category1 = Category.objects.create(
            name           = 'Test1',
            icon_image_url = 'test_icon.url1'
        )
        category2 = Category.objects.create(
            name           = 'Test2',
            icon_image_url = 'test_icon.url2'
        )
        category3 = Category.objects.create(
            name           = 'Test3',
            icon_image_url = 'test_icon.url3'
        )
```
테스트 함수에도 변수 이름은 명확해야 합니다 지원님 :)
아래 다른 코드에서도 변수명 예쁜걸로 바꿔주세요~ 👍

Reply:
데이터 베이스 문제 해결하니까 변수가 필요없어졌어요!!